api/package.json

```diff
@@ -2,11 +2,13 @@
   "name": "api",
   "version": "1.0.0",
   "description": "",
-  "main": "server.js",
+  "main": "src/server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon --exec babel-node server.js",
-    "start-local": "source .env.bash && nodemon --exec babel-node server.js"
+    "start": "nodemon --exec babel-node src/server.js",
```

Comment:
Should this be `nodemon ...` or `node ...` because the error that comes up on the heroku console is `sh: nodemon not found`

Reply:
Yeah don't worry because Heroku now run `yarn run serve` so this script is now for local use